com.unity.film-tv.toolbox/Samples/MultiScene/samples.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
**.sample.json** not **samples.json** :)
The way to know if this works, is if you see the samples button in the packman ui (look the way USD looks)

Reply:
got it - pushed up a commit with the rename